export/pivot-table/README.MD

```
@@ -0,0 +1,122 @@
+`listener.on('job:ready', { job: 'workbook:generatePivotTable' })`
+
+<!-- END_INFOCARD -->
+
+
+> When embedding Flatfile, this plugin should be deployed in a server-side listener. [Learn more](/docs/orchestration/listeners#listener-types)
+
+
+
+## Parameters
+
+#### `pivotColumn` - `string` 
```

Markdownlint

Expected: h3; Actual: h4
Heading levels should only increment by one level at a time

(MD001, heading-increment)